deployments/ocean/image/binder/dask_config.yaml

```diff
@@ -21,18 +21,18 @@ kubernetes:
           - '2'
           - --no-bokeh
           - --memory-limit
-          - 12GB
+          - "11.5GB"
           - --death-timeout
           - '60'
         image: ${JUPYTER_IMAGE_SPEC}
         name: dask-${JUPYTERHUB_USER}
         resources:
           limits:
             cpu: "1.75"
```

Comment:
I think this could be allowed to be be as high as the node you schedule on can use. It is an optimization without drawbacks as I understand it.

Reply:
I'm a bit confused about the interaction between dask and kubernetes here. Each has its own independent mechanisms to manage CPU and memory usage. The dask worker process here is limited to 2 threads. It has no notion of "requests" vs "limits". 

In any case, i will try following your advice.

Reply:
Ah I've been very uncertain about how a dask-worker consume resources. Is it correct that since since dask the dask workers are limited to 2 threads, they will never use more than 2 CPU?

Even though they are managed by linux cgroups etc, I think of these Kubernetes settings as restricting the pods resources and that they can be seen as a description of the dask-workers CPU/Memory usage restrictions, but they will also be used to influence how many dask workers can be fit onto a node and how much of the nodes total CPU/Memory will end up being in active use.